package.json

```diff
@@ -100,6 +100,6 @@
     "stylelint-suitcss": "^1.0.0",
     "terra-toolkit": "^2.1.0",
     "webpack": "^3.6.0",
-    "webpack-dev-server": "2.7.1"
+    "webpack-dev-server": "^2.9.1"
```

Comment:
We need to keep this locked down for ie 10 & 11 support.

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/b6a950329f831a533a9f38d54427d9c17a5e17e6#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R102